### DISCUSSION TITLE
Automatic associate UpdateToken script during dynamic registration

### DISCUSSION THREAD

**Discussion by tawaren** [JanssenProject/jans#7315] [Q&A] [answered]
We have decided to use Janssen for a project, and I have successfully written some script for a proof of concept to show that our use case can be implemented with Janssen. However, one of the scripts needed is an UpdateToken, which every client with a certain Scope needs to use. Otherwise, the security of our use case is impacted. 

First, I tried to add the UpdateToken with a DynamicClientRegistration Script but could not figure out how and if it is possible.
Second, I tried to make an UpdateToken that applies to all clients, which, according to documentation, should be possible, but I could not figure out how to make it work.

I would appreciate a hint in the right direction on how to make it work because otherwise, we need to disable dynamic registration and add clients manually.

**Comment by yuriyz** [comment, marked as answer]
You can look to swagger to find the right properties. In this particular case it is `update_token_script_dns`. (Note it's array that accepts script DNs, not names).

https://github.com/JanssenProject/jans/blob/afc1636efeb3803e851e63f44b35a69583d5931c/jans-auth-server/docs/swagger.yaml#L1615

**Reply by tawaren** [reply]
Thanks for the hint, I got it working.
